README.md

```diff
@@ -1,37 +1,44 @@
 # JUMP-single-cell
 
-Single cell analysis of the JUMP Cell Painting consortium data.
-Please ensure adequate storage as the sqlite plate data is approximately 1.1 TB large.
+## Data
 
-## 0.download_data
+In this repository, we apply the [phenotypic profiling model](https://github.com/WayScience/phenotypic_profiling_model), which predicts phenotypic class of single cells using nuclei features, on the [JUMP-Target pilot data](https://github.com/jump-cellpainting/JUMP-Target) from the [JUMP consortium](https://jump-cellpainting.broadinstitute.org/).
 
-This folder contains the code and metadata file to download the plate data from aws.
-For more information on how the plate paths were generated please review the `0.generate_jump_dataset_manifest.ipynb` file.
+In this dataset, there are 51 plates that are associated with one of three treatment types:
 
-To download the data, first run `0.generate_jump_dataset_manifest.ipynb`, to generate the paths data, then run `1.download_jump_single_cell_profiles.ipynb` to download the plate data.
+- Compound
+- CRISPR
+- ORF
 
-## 1.process_data
+Each treatment has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
+A barcode platemap is include which associates each plate to the correct platemap file.
 
-In this step, we merged the single cell data using pycytominer, and the reference plate data located in the `reference_plate_data` folder in the root of the git directory.
-The metadata files in this folder can be located by following this aws link:
-`https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0000-jump-pilot/source_4/workspace/metadata/external_metadata/`
+There are a total of **20,959,860 single cells** between all plates.
```

Comment:
```suggestion
We segment a total of **20,959,860 single cells** in all plates.
```

Different segmentation algorithms and/or parameters may disagree with this number :)

Reply:
I might be confused by this one because when I think "we segment" that means we perform the CellProfiler pipeline but that is probably just semantics. I have implemented this change.